internal/config/config.go

```diff
@@ -32,6 +32,11 @@ type Config struct {
 		Extension struct {
 			Enabled bool `envconfig:"default=true"`
 		}
+		Creator struct {
+			Enabled     bool   `envconfig:"default=true"`
+			FgaGrpcAddr string `envconfig:"required"`
+			StoreId     string `envconfig:"default=1"`
```

Comment:
I would make StoreId required, for me it is on the same level of importance as FgaGrpcAddr - without it account operator will work incorrectly.